src/SFA.DAS.EAS.Employer_Financial.Database/Scripts/Manual/AML-2502_DeleteFullyFundedPayments.sql

```diff
@@ -0,0 +1,6 @@
+﻿DELETE meta FROM [employer_financial].[PaymentMetaData] meta
+INNER JOIN [employer_financial].[Payment] p ON p.PaymentMetaDataId = meta.Id
+WHERE p.FundingSource = 4
+
+DELETE FROM [employer_financial].[Payment]
```

Comment:
I think payments will need to be deleted before meta data since the foreign key is on the Payments table.